.github/workflows/cd.yaml

```
@@ -0,0 +1,19 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+
+name: cd
+on:
+  workflow_dispatch:
+#  check_run:
+#    types:
+#      - completed
```

```suggestion
  check_run:
    types:
      - completed
```
This is needed for CD.